src/Equinor.ProCoSys.Completion.WebApi/Controllers/Misc/CacheController.cs

```
@@ -88,4 +96,21 @@ public async Task<IList<AccessablePlant>> GetPlantsFromMain()
         var plants = await _permissionApiService.GetAllPlantsForUserAsync(currentUserOid);
         return plants;
     }
+
+    [HttpGet("PrefetchCheckListByGuid/{checkListGuid}")]
+    public async Task<ActionResult> PrefetchCheckListByGuid(
```

anonymous?

---

A bit unsure about that, from frontend perspective, they should ignore any errors, as they are not relevant for them for this endpoint. And it is not returning any data.

---

Front end does not need to care what we return, but the API should behave as expected. Next time someone uses the endpoint, maybe they do care.